<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HetGetClass</class>
 <widget class="QDialog" name="HetGetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>531</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Input File</string>
   </property>
   <widget class="QLineEdit" name="infile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseInFile">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>531</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QCheckBox" name="nlCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>241</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Tape is Non-Labeled</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="inputBlocksSpinBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>25</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File Number</string>
    </property>
   </widget>
   <widget class="QComboBox" name="recfmComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>9</number>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FBA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FBM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FBS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FBSA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FBSM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FSA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FSM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>U</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VBA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VBM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VBS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VBSA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VBSM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VSA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VSM</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>65</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Recfm</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="lreclSpinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>32767</number>
    </property>
    <property name="value">
     <number>80</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>65</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lrecl</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="blksizeSpinBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>32768</number>
    </property>
    <property name="value">
     <number>1600</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>65</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Blksize</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="asciiCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>421</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert Output To ASCII (Implies Unblock)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="unblockCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>421</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Unblock (Removes BDWs And RDWs If RECFM=V)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="stripCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>431</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Strip Trailing Blanks (requires Convert To Ascii)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>531</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Output File</string>
   </property>
   <widget class="QLineEdit" name="outdir">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseOutDir">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outfile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseOutFile">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Dir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
